profiles/non_cern_machine/chamberInfo.py

```diff
@@ -0,0 +1,225 @@
+"""
+``chamberInfo`` --- Information about installed chambers
+--------------------------------------------------------
+"""
```

Comment:
I think this should stay with the `gem-plotting-tools` repo.  Later some `user_constants` will be addressed by [this comment](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/189#issuecomment-472372133)

Reply:
so undo this commit if possible (or `git rm` this file).

Reply:
File removed from commit.